packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,107 @@
+
+For more specific use-cases for exchanges, please checkout the [Custody and Exchange Integration Guide](https://docs.celo.org/developer-guide/overview/integrations/custody) as well.
+
+## Important Information
+### Celo Native Asset and Celo Dollar
+
+There are two important assets on the Celo network, the Celo native asset (CELO) and the Celo Dollar (cUSD). CELO was called Celo Gold (cGLD) when the contract was deployed, so you will often see references to Celo Gold in the codebase. To learn more about the two, please read [this](https://docs.celo.org/developer-guide/overview/celo-for-eth-devs#the-celo-native-asset-and-the-celo-dollar) section of the docs.
+
+## Resources
+### Address for CELO and cUSD
+The smart contract address for CELO can be found [here](https://explorer.celo.org/address/0x471ece3750da237f93b8e339c536989b8978a438/transactions).
+The smart contract address for cUSD can be found [here](https://explorer.celo.org/address/0x765de816845861e75a25fca122bb6898b8b1282a/transactions).
+
+
+### Useful API endpoints
+The following are useful API endpoints available to you that would help you in your listings of the CELO and cUSD digital assets.
+
+#### CELO and cUSD
+For querying the API on total coins in circulation in CELO and cUSD, the following endpoint will provide you with that:
+```sh
+$ curl  [https://thecelo.com/api/v0.1.js?method=ex_totalcoins](https://thecelo.com/api/v0.1.js?method=ex_totalcoins) {"code":"200","msg":"success","data":{"CELO":608485841.9959723,"cUSD":10250632.56099673}}
+```
+
+##### cUSD Circulating Supply
+Circulating Supply refers to the # of coins that are circulating in the market and in the general public's hands.
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_cusd_circulating
+11353464.550486518
+```
+
+#### CP-DOTO 
+
+For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
+
+##### Summary
+Summary overview.
+
+```sh 
+$ curl https://thecelo.com/api/v0.1.js?method=ex_summary
+
+{"trading_pairs":"CELO_CUSD","last_price":2.6143,"lowest_ask":2.5933609958506225,"highest_bid":2.5676,"base_volume":37524.32000000003,"quote_volume":14714.520000000002,"price_change_percent_24h":3.7027120070382127,"highest_price_24h":2.649,"lowest_price_24h":2.4787}}
+```
+
+##### Assets
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_assets
+
+{"code":"200","msg":"success","data":{"CELO":{"name":"CELO","unified_cryptoasset_id":"5567","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"},"CUSD":{"name":"Celo Dollars","unified_cryptoasset_id":"825","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"}}}
+```
+
+##### Ticker
```

Comment:
Might want to add description: "24-hour rolling window price change statistics for all markets"